tests/e2e/manifests/chit/tpl-clickhouse-22.8.yaml

```diff
@@ -13,5 +13,5 @@ spec:
         spec:
           containers:
             - name: clickhouse-pod
-              image: clickhouse/clickhouse-server:22.8
+              image: clickhouse/clickhouse-server:23.8
```

Comment:
rollback it

Reply:
add separate `tpl-clickhouse-23.8.yaml`

and look references to tpl in `tests`

Reply:
As far as I can see the file is already present: https://github.com/Altinity/clickhouse-operator/blob/master/tests/e2e/manifests/chit/tpl-clickhouse-23.8.yaml